src/main/java/wooteco/subway/line/LineService.java

```diff
@@ -0,0 +1,48 @@
+package wooteco.subway.line;
+
+import org.springframework.stereotype.Service;
+import wooteco.subway.exception.LineNameDuplicatedException;
+import wooteco.subway.exception.LineNotFoundException;
+import wooteco.subway.line.dao.LineDao;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class LineService {
+
+    private final LineDao lineDao;
+
+    public LineService(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    public Line createLine(String name, String color) {
```

Comment:
```suggestion
    public Line create(String name, String color) {
```
line 서비스니까 당연히 line에 대한 처리를 하겠죠? 메서드에 중복해서 써줄 필요는 없습니다.

Reply:
이외에 다른  메소드 이름도 정리해주었어요!